src/main/java/uk/gov/justice/probation/courtcasematcher/messaging/HearingProcessor.java

```diff
@@ -123,13 +123,12 @@ private void updateAndSave(final Hearing hearing) {
     }
 
     Hearing assignUuids(Hearing hearing) {
-        // Apply the new case ID
-        final var caseId = UUID.randomUUID().toString();
-        var updatedHearing = hearing.withCaseId(caseId).withHearingId(caseId);
+        var updatedHearing = hearing.withHearingId(UUID.randomUUID().toString());
 
         // We want to retain the LIBRA case no if present
         if (hearing.getCaseNo() == null) {
```

Comment:
I think this is using the builder patter to create a new `Hearing` called `updatedHearing` which is the same as `hearing` apart from adding a random `hearingId`?

If so, might be less confusing to use this `updatedHearing` from this point onwards?

Reply:
Yes, we no longer need to generate caseId but still need to generate a hearingId